.gitignore

```diff
@@ -16,4 +16,5 @@ TestResults
 !.vscode/
 .vscode/*
 !.vscode/launch.json
-!.vscode/tasks.json
\ No newline at end of file
+!.vscode/tasks.json
```

Comment:
So are we keeping this or not?

Reply:
I think so. I'm just adding an CR\LF to add the next entry (.DS_Store)